pallets/runtime/src/bridge.rs

```diff
@@ -0,0 +1,376 @@
+        /// The multisig account of the bridge validator set. The genesis signers must accept their
+        /// authorizations to be able to get their proposals delivered.
+        Validators get(validators) build(|config: &GenesisConfig| {
+            if config.signatures_required > u64::try_from(config.signers.len()).unwrap_or_default()
+            {
+                panic!("too many signatures required");
+            }
+            if config.signatures_required == 0 {
+                /// Default to the empty validator set.
+                return Default::default();
+            }
+            <multisig::Module<T>>::create_multisig_account(
+                Default::default(),
+                config.signers.as_slice(),
+                config.signatures_required
+            ).expect("cannot create the bridge multisig")
+        }): T::AccountId;
+        /// Correspondence between validator set change proposals and multisig proposal IDs.
+        ValidatorSetProposals get(validator_set_proposals): map ValidatorSet => Option<u64>;
+        /// Correspondence between bridge transaction proposals and multisig proposal IDs.
+        BridgeTxProposals get(bridge_tx_proposals): map BridgeTx<T::AccountId> => Option<u64>;
+        /// Pending issuance transactions to identities.
+        PendingTxs get(pending_txs): map IdentityId => Vec<BridgeTx<T::AccountId>>;
+    }
+    add_extra_genesis {
+        /// The set of initial validators from which a multisig address is created at genesis time.
+        config(signers): Vec<Signatory>;
+        /// The number of required signatures in the genesis validator set.
+        config(signatures_required): u64;
+    }
+}
+
+decl_event! {
+    pub enum Event<T> where AccountId = <T as frame_system::Trait>::AccountId {
+        /// Confirmation of a validator set change.
+        ValidatorSetChanged(ValidatorSetChange<AccountId>),
+        /// Confirmation of minting POLY on Polymesh in return for the locked ERC20 tokens on
+        /// Ethereum.
+        Bridged(BridgeTx<AccountId>),
+        /// Notification of an approved transaction having moved to a pending state due to the
+        /// recipient identity either being non-existent or not having a valid KYC.
+        Pending(PendingTx<AccountId>),
+        /// Notification of a failure to finalize a pending transaction. The transaction is removed.
+        Failed(BridgeTx<AccountId>),
+    }
+}
+
+decl_module! {
+    pub struct Module<T: Trait> for enum Call where origin: T::Origin {
+        type Error = Error<T>;
+
+        fn deposit_event() = default;
+
+        /// Proposes to change the bridge validators and the number of required signatures, which
+        /// amounts to making a multisig proposal for the validator set change if the change is new
+        /// or approving an existing proposal if the change has already been proposed.
+        pub fn propose_validator_set(origin, validator_set: ValidatorSet) ->
```

Comment:
I still don't see the requirement of this function. Bridge validators can directly call the create_proposal function in the multisig module.